.moban.yaml

```diff
@@ -59,6 +60,7 @@ targets:
   - .editorconfig: .editorconfig.jj2
   - setup.py: core-setup.py.jj2
   - setup.cfg: core-setup.cfg.jj2
+  - coalib/misc/Asyncio.py: core-setup.cfg.jj2
```

Comment:
this is wrong

Reply:
Here you are replacing a .py file with garbage.

You need to use moban variable `nocover_file_globs`